### ISSUE TITLE
Fix whisper tests and add a configuration test for Rinkeby network

### ISSUE THREAD

**Issue by farazdagi** [status-im/status-go#150]
- do after #143 (as it depends on it)
- https://www.rinkeby.io/
- make sure `GeneretaConfig()` exposed sane defaults for the net=4
- revisit default parameters (`DatabaseCache` in particular) - to make sure that we are running fine-tuned embedded node (as outlined in https://www.rinkeby.io/)
- `status faucet` needs to support both Ropsten and Rinkeby (probably deserves a separate ticket)
- update CHT

**Comment by farazdagi** [comment]
Regarding CHT setting, @karalabe pointed out:
Clique will need a bit custom CHT
OR specifically, you'll only need to set 1 single hash for it
but that can only be for an epoch block
(every 30K)

**Comment by influx6** [comment]
We need to update current version of `web3.js` in use to resolve issues with test cases in #241.

**Comment by tiabc** [comment]
Clear requirements for the issue:
1. Write Rinkeby configuration test.
2. Fix whisper tests caused by rebase onto geth1.6.7.
